Updated text + filled in blanks (#2)
greenelab/nature_news_manuscript: content/03.methods.md

Using an additional spider defined in “doi_crawl.py”, we scaped all citations within news articles.
For simplicity, we only considered citations with a DOI included in either text or a hyperlink in this spider.
Other possible forms of citations, e.g., titles, were not included.
The DOIs were then queried using the _Springer_ API.
The spider “article_author_crawl.py” scraped all articles designated "Article" or "Letters" from all possible research articles.
We only scraped author names, author positions, and associated affiliations from research articles.
It should be noted that news article designations changed over time.

#### coreNLP

After news articles were scraped and processed, the text was processed using the coreNLP pipeline [@doi:10.3115/v1/P14-5010] (version 4.2.0).
The main purpose for using coreNLP was to identify named entities related to countries and quoted speakers.
The full set of annotaters were: tokenize, ssplit, pos, lemma,ner, parse, coref, quote.
We used the "statistical" algorithm to perform coreference resolution.
All results were output to json format for further downstream processing.

#### _Springer_ API

> _Springer_ was chosen over other publishers for multiple reasons: 1) it is a large publisher, second only to Elsevier; 2) it covers multiple subjects, in contrast to PubMed; 3) it has a large daily query limit (5000/day); and 4) it provided more author affiliation information than found in Elsevier. 

Reviewer: ```suggestion
_Springer_ was chosen over other publishers for multiple reasons: 1) it is a large publisher, second only to Elsevier; 2) it covers multiple subjects, in contrast to PubMed; 3) its API has a large daily query limit (5000/day); and 4) it provided more author affiliation information than found in Elsevier. 
```

We selected a random set of articles to generate the _Springer_ background set.
These articles were the first 200 English language "Journal" articles returned by the _Springer_ API for each month, resulting in 2400 articles per year for 2005 through 2020.
To get the author information for the cited articles, we queried the _Springer_ API using the scraped DOI.
For both API query types, the author names, positions, and affiliations for each publication were stored and are available in "all_author_country.tsv" and "all_author_fullname.tsv".
 
### Name Normalization
 
Names were cast to lowercase and processed using the R package humaniformat [@https://cran.r-project.org/web/packages/humaniformat/index.html].
humaniformat identifies if names are reversed (Lastname, Firstname), as well as identifies middle names.
Since many last or first authorships may be non-names, we additionally filtered out any identified names if they partially or fully match any of the following terms: "consortium", "group", "initiative", "team", "collab", "committee", "center", "program", "author", or "institute".

> Furthermore, since many articles only contain first and last name initials, we remove any names with a  "." or "-" and are less than four characters.

Reviewer: ```suggestion
Furthermore, since many articles only contain first and last name initials, we remove any names with a  "." or "-" and those less than four characters.
```

> Furthermore, since many articles only contain first and last name initials, we remove any names with a  "." or "-" and are less than four characters.

Reviewer: ```suggestion
Furthermore, since many articles only contain first and last name initials, we remove any names less than four letters with a  "." or "-".
```

Reviewer: I think one of these is right, but I wasn't sure which one so I provide two suggestions 😁

Author: yes, this is the correct one
